code/game/objects/items/food/pizza.dm

```diff
@@ -390,3 +404,12 @@
 	icon_state ="energypizzaslice"
 	tastes = list("pure electricity" = 4, "pizza" = 2)
 	foodtypes = TOXIC
+
+///screentips for pizza
+/obj/item/food/pizza/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+
+	if(held_item?.tool_behaviour == TOOL_KNIFE || TOOL_SCALPEL || TOOL_SAW)
+		if(slice_type == null)
+			return
+		context[SCREENTIP_CONTEXT_LMB] = "Slice pizza"
+		return CONTEXTUAL_SCREENTIP_SET
```

Comment:
i will look into the feasibility of (me personally) trying to add contextual screentips to the element because your solution wrt tools is what i was hoping i would not have to do haha. 

thank you!

Reply:
it turns out that someone WAS me! and only with medium help from @Jacquerel !!!